certora/specs/HashConsensus.spec

```diff
@@ -0,0 +1,814 @@
+// using AccountingOracle as AccountingOracleContract
+
+
+/**************************************************
+ *               Methods Declaration              *
+ **************************************************/
+methods {
+    submitConsensusReport(bytes32 report, uint256 refSlot, uint256 deadline) => NONDET
+    getLastProcessingRefSlot() returns (uint256) => NONDET
+    getConsensusVersion() returns (uint256) => NONDET
+}
+
+/**************************************************
+ *                CVL FUNCS & DEFS                *
+ **************************************************/
+function saneTimeConfig() {
```

Comment:
Why do you declare the env variable inside the function, and not use it as an input variable?